cpp/velox/benchmarks/ColumnarToRowBenchmark.cc

```diff
@@ -15,153 +15,54 @@
  * limitations under the License.
```

Comment:
This benchmark can be deleted because Velox has a similar benchmark, and we only call the velox API